src/features/defi/components/Approve/Approve.tsx

```diff
@@ -36,9 +36,8 @@ type ApproveProps = {
   fiatEstimatedGasFee: string
   isExactAllowance?: boolean
   isApproved?: boolean
-  learnMoreLink?: string
```

Comment:
Also effectively useless - always the same link and same thing, it not being omitted was a bug, not a feature.